### ISSUE TITLE
bug: commit confirm window in neogit shows nothing

### ISSUE THREAD

**Issue by TroySigX** [folke/noice.nvim#388]
### Did you check docs and existing issues?

- [X] I have read all the noice.nvim docs
- [X] I have searched the existing issues of noice.nvim
- [X] I have searched the exsiting issues of plugins related to this issue

### Neovim version (nvim -v)

0.8.3

### Operating system/version

6.2.2-arch1-1

### Describe the bug

When the commit confirm window of neogit shows up, it was blank.
![image](https://user-images.githubusercontent.com/88051085/224358095-87bee457-6bf1-45d5-9690-086499239cce.png)

### Steps To Reproduce

1. commit something in neogit
2. `:wq`
3. Confirm window pops up and it shows nothing

### Expected Behavior

Yes and no options should appear on the window.

**Comment by TroySigX** [comment]
Here is my config:
```
local lazypath = vim.fn.stdpath('data') .. '/lazy/lazy.nvim'
if not vim.loop.fs_stat(lazypath) then
    vim.fn.system({
        'git',
        'clone',
        '--filter=blob:none',
        'https://github.com/folke/lazy.nvim.git',
        '--branch=stable',
        lazypath,
    })
end
vim.opt.rtp:prepend(lazypath)

require('lazy').setup({
    {
        'TimUntersberger/neogit',
        dependencies = {
            'sindrets/diffview.nvim',
            'nvim-lua/plenary.nvim'
        },
        config = function()
            require('neogit').setup {
                disable_signs = false,
                disable_hint = false,
                disable_context_highlighting = false,
                disable_commit_confirmation = false,
                auto_refresh = true,
                disable_builtin_notifications = false,
                use_magit_keybindings = false,
                kind = "tab",
                console_timeout = 2000,
                auto_show_console = true,
                popup = {
                    kind = "vsplit",
                },
                signs = {
                    -- { CLOSED, OPENED }
                    section = { ">", "v" },
                    item = { ">", "v" },
                    hunk = { "", "" },
                },
                integrations = {
                    diffview = true
                },
                sections = {
                    untracked = {
                        folded = false
                    },
                    unstaged = {
                        folded = false
                    },
                    staged = {
                        folded = false
                    },
                    stashes = {
                        folded = true
                    },
                    unpulled = {
                        folded = true
                    },
                    unmerged = {
                        folded = false
                    },
                    recent = {
                        folded = true
                    },
                },
            }
        end,
    },
    {
        'folke/noice.nvim',
        dependencies = {
            'MunifTanjim/nui.nvim',
        },
        config = function()
            require('noice').setup({
                cmdline = {
                    view = 'cmdline',
                },
                lsp = {
                    override = {
                        ['vim.lsp.util.convert_input_to_markdown_lines'] = true,
                        ['vim.lsp.util.stylize_markdown'] = true,
                        ['cmp.entry.get_documentation'] = true,
                    },
                },
                presets = {
                    bottom_search = true,
                    command_palette = false,
                    long_message_to_split = true,
                    inc_rename = true,
                    lsp_doc_border = false,
                },
            })
        end,
    },
})
```

**Comment by TroySigX** [comment]
Hi, is there any update on this?

**Comment by folke** [comment]
It you want to look into this or create a pr, go ahead. I don't use neogit, so not planning to look into this.

**Comment by TroySigX** [comment]
I dont know how to do it, can anyone please help?

**Comment by vandalt** [comment]
I think this is a duplicate of #232, which is fixed by TimUntersberger/neogit#441. As mentioned there, you can use `disable_commit_confirmation = true` in the meantime, so the pop-up won't appear at all.
